app/components/GroupTasks.js

```diff
@@ -1,19 +1,31 @@
 import React from 'react'
-import { StyleSheet, View, Text, Button } from 'react-native'
+import { StyleSheet, View, Text, Button, ListView } from 'react-native'
```

Comment:
Consider making long lists alphabetical.  Will always still approve, but might make for nicer code and is very easy.